payments/build.gradle

```diff
@@ -1,14 +1,15 @@
-plugins {
-    id 'com.android.library'
-    id 'kotlin-android'
+apply from: configs.androidLibrary
 
-    id 'signing'
-    id 'maven-publish'
-}
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
```

Comment:
You shouldn't need these two, they're already part of the parent `configs.androidLibrary` config file.

Reply:
You can also remove the entire `android {}` closure as all that configuration is also part of the parent config file.

Reply:
Also, this adds detekt as well : )